<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="UMESH" failures="2" tests="8" name="Test2" time="53.323" errors="0" timestamp="2024-01-06T12:01:42 IST">
  <testcase classname="pages.test.PageTest3" name="test1_PageTest3" time="0.439"/>
  <testcase classname="pages.test.PageTest3" name="test2_PageTest3" time="0.006"/>
  <testcase classname="pages.test.PageTest3" name="test3_PageTest3" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at pages.test.PageTest3.test3_PageTest3(PageTest3.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3_PageTest3 -->
  <testcase classname="pages.test.PageTest3" name="test4_PageTest3" time="0.004"/>
  <testcase classname="pages.test.PageTest3" name="test5_PageTest3" time="0.004"/>
  <testcase classname="pages.test.PageTest2" name="test1" time="0.011"/>
  <testcase classname="pages.test.PageTest2" name="test3" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at pages.test.PageTest2.test3(PageTest2.java:20)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="pages.test.PageTest2" name="test4" time="0.021"/>
</testsuite> <!-- Test2 -->
